libcnb-data/src/buildpack/mod.rs

```diff
@@ -461,22 +486,26 @@ checksum = "abc123"
     #[test]
     fn deserialize_minimal_component_buildpack() {
         let toml_str = r#"
-api = "0.9"
+api = "0.10"
 
 [buildpack]
 id = "foo/bar"
 version = "0.0.1"
 
-[[stacks]]
-id = "*"
+[[targets]]
+os = "linux"
+arch = "amd64"
```

Comment:
It could be, I added this in since the test previously also asserted against the `buildpack_descriptor.targets` field. We can change this to asserting if that field is empty. Works for me either way.

Reply:
1f9d849